docs/2023_Tech.md

```diff
@@ -14,6 +14,8 @@ The logbook must be present with the vehicle and presented at Technical Inspecti
 
 All required personal equipment must be in good condition and have proper approval markings from SFI or FIA. All required safety equipment shall be used and installed per manufacturer’s recommendation.
 
+/*  This is Paul trying to edit something  */
```

Comment:
I do not like this change. There is too much mayonnaise in it.

Reply:
But mayonnaise is good for you.  Way better than ranch dressing.

Reply:
But you can only use Russian on a good Ruben.